js/src/OrdinalScaleModel.ts

```diff
@@ -27,7 +27,7 @@ export class OrdinalScaleModel extends ScaleModel {
   }
 
   typedRange(values) {
-    return new Int32Array(values.map(Number));
+    return values;
```

Comment:
As per @bollwyvl 's recommendation, this could be:

```typescript
return new Int32Array(values.map(this.domain.indexOf, this.domain));
```

Which would make the scale work with the indeces of the `domain` (I believe that is what is meant by range [here](https://github.com/bqplot/bqplot/pull/1226#discussion_r509214964), @maartenbreddels please don't hesitate to let us know if we misunderstood what was intended by `range`).

We went with the `values` instead because otherwise it could break other people's code, but we acknowledge it would probably be better to use the line above.